corehq/apps/export/const.py

```diff
@@ -6,6 +6,12 @@
 PROPERTY_TAG_INFO = 'info'
 PROPERTY_TAG_UPDATE = 'update'
 PROPERTY_TAG_SERVER = 'server'
+PROPERTY_TAG_DELETED = 'deleted'
+PROPERTY_TAG_ROW = 'row'
+
+MAIN_TABLE_PROPERTIES = [
```

Comment:
only 1 for right now, but this is how we can add props to the main table that aren't questions